RefactoringPracticeI/GameInventory/Inventory/Item.cs

```diff
@@ -1,11 +1,15 @@
-﻿namespace RefactoringExerciseI.Inventory
+﻿
+
+namespace RefactoringExerciseI.Inventory
 {
-    public class Item
+    public abstract class Item
```

Comment:
clever, also you could extend this class and add your new method in the new class that extends this one (kind of a decorator) but this still works. No need to do anything